helper/test_helpers.go

```diff
@@ -0,0 +1,33 @@
+package helper
+
+import (
+	abci "github.com/tendermint/tendermint/abci/types"
+)
+
+type TestOpts struct {
+	app     abci.Application
+	chainId string
+}
+
+func NewTestOpts(app abci.Application, chainId string) *TestOpts {
+	return &TestOpts{
+		app:     app,
+		chainId: chainId,
+	}
+}
+
+func (t *TestOpts) SetApplication(app abci.Application) {
+	t.app = app
+}
+
+func (t *TestOpts) GetApplication() abci.Application {
```

Comment:
`GetApplication` and `SetChainId` are not used, right? Maybe we can remove them

Reply:
We can but it looks complete to have both sets of getters and setters. Besides, it might be used later in other tests.